aioxmpp/xso/model.py

```diff
@@ -2684,3 +2684,126 @@ def processingInstruction(self, target, data):
 
     def skippedEntity(self, name):
         self.__receiver.skippedEntity(name)
+
+
+class XSOEnumMixin:
+    """
+    Mix-in to create enumerations of XSOs.
+
+    .. versionadded:: 0.10
+
+    The enumeration member values must be pairs of ``namespace``, ``localpart``
+    strings. Each enumeration member is equipped with an :attr:`xso_class`
+    attribute at definition time.
+
+    .. automethod:: to_xso
+
+    .. autoattribute:: enum_member
+
+    .. attribute:: xso_class
+
+        A :class:`aioxmpp.xso.XSO` *subclass* which has the enumeration members
+        value as :attr:`~.XSO.TAG`. So the subclass matches elements which have
+        the qualified tag in the enumeration member value.
+
+        The class does not have any XSO descriptors assigned. They can be added
+        after class definition.
+
+        .. attribute:: enum_member
+
+            The enumeration member to which the :attr:`xso_class` belongs.
+
+            This allows to use XSOs and enumeration members more exchangably;
```

Comment:
exchangably => interchangeably (or at least exchangeably, according to wiktionary this is the correct spelling).